src/data_collection.py

```diff
@@ -179,6 +179,7 @@ def calculate_individual_metrics(json_file_name):
         for key in current_data["RAW_DATA"]:
             author = key["author_name"]
             email = key["author_email"]
+            filepaths = key["filepath"]  # get filepaths of file modified
```

Comment:
I assume this is for checking if the filepath is to the 'tests' folder.

Reply:
Yes.